### PR TITLE
Feature/expected output amount

### PR DESCRIPTION
**Pull request by bryzettler**

### CODE REVIEW

File: packages/react/src/components/Swap/Royalties.tsx [hunk 1]
```diff
@@ -19,14 +19,21 @@ export function Royalties({
   isBuying: boolean;
   formRef: React.MutableRefObject<HTMLInputElement>;
 }) {
+  const { metadata: baseMeta, loading: baseMetaLoading } = useTokenMetadata(
+    tokenBonding.baseMint
+  );
+
+  const { metadata: targetMeta, loading: targetMetaLoading } = useTokenMetadata(
```
Comment (ChewingGlass) [inline packages/react/src/components/Swap/Royalties.tsx:26]: Are baseMetaLoading and targetMetaLoading used anywhere?

File: packages/react/src/components/Swap/Swap.tsx [hunk 2]
```diff
@@ -7,49 +7,50 @@ import { Notification } from "../Notification";
...
+  const { loading: driverLoading, ...swapProps } = useSwapDriver({
+    tradingMints,
+    onTradingMintsChange: setTradingMints,
+    swap: (args) =>
+      execute(args).then(({ targetAmount }) => {
+        toast.custom((t) => (
+          <Notification
+            show={t.visible}
+            type="success"
+            heading="Transaction Succesful"
```
Comment (ChewingGlass) [inline packages/react/src/components/Swap/Swap.tsx:41]: *Successful

File: packages/react/src/hooks/useMetaplexMetadata.ts [hunk 3]
```diff
@@ -40,6 +43,7 @@ const solMetadata = new MetadataData({
  * @param token
  * @returns
  */
+const metadatas: any = {};
```
Comment (ChewingGlass) [inline packages/react/src/hooks/useMetaplexMetadata.ts:46]: Leftover from debugging?

**Review by ChewingGlass — CHANGES_REQUESTED** [review]

File: packages/react/src/components/Swap/Royalties.tsx (hunk 1 above)
Comment (bryzettler) [reply packages/react/src/components/Swap/Royalties.tsx:26]: Looks like no

File: packages/react/src/hooks/useMetaplexMetadata.ts (hunk 3 above)
Comment (bryzettler) [reply packages/react/src/hooks/useMetaplexMetadata.ts:46]: you

**Review by ChewingGlass — APPROVED** [review]

**Pull request merged**